### PR TITLE
Feature/crud admin user

### PR DESCRIPTION
**Pull request by LeandroSoria2**
### What changes does this PR include?
**Description**
Add active admin gem and set up active admin and add  sass rails gem
**Summary**
-Add to gem file active admin gem
-Add to gem file active admin gem
-Change application_controller
**Trello**
(https://trello.com/c/9narEXDz/30-3-as-an-admin-i-should-be-able-to-login-in-the-admin-page)

### CODE REVIEW

File: db/migrate/20220701173654_create_active_admin_comments.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      t.references :resource, polymorphic: true
+      t.references :author, polymorphic: true
+      t.timestamps
+    end
+    add_index :active_admin_comments, [:namespace]
+  end
+
+  def self.down
+    drop_table :active_admin_comments
+  end
```
Comment (JulianPasquale) [inline db/migrate/20220701173654_create_active_admin_comments.rb]: Can we define the `change` method here?

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,7 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+if Rails.env.development?
```
Comment (JulianPasquale) [inline db/seeds.rb]: Seeds are not meant to be run in production usually. I think we can remove this condition

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,3 +1,2 @@
 class ApplicationController < ActionController::Base
-  include DeviseTokenAuth::Concerns::SetUserByToken
```
Comment (JulianPasquale) [inline app/controllers/application_controller.rb:2]: Why is this being removed? Don't we need to add it somewhere else?

File: db/migrate/20220701173654_create_active_admin_comments.rb (hunk 1 above)
Comment (LeandroSoria2) [reply db/migrate/20220701173654_create_active_admin_comments.rb]: It was generated by active_admin, but I will change it.

File: db/seeds.rb (hunk 2 above)
Comment (LeandroSoria2) [reply db/seeds.rb]: I think it's better to keep this validation just in case someone accidentally runs this on production, wdyt?

File: app/controllers/application_controller.rb (hunk 3 above)
Comment (LeandroSoria2) [reply app/controllers/application_controller.rb:2]: I have a problem with active admin and this is already in the API controller.

https://github.com/LeandroSoria2/TargetApi/blob/develop/app/controllers/api/v1/api_controller.rb

Comment (AgostinaDev) [reply app/controllers/application_controller.rb:2]: @JulianPasquale It looks like including this line in the ApplicationController generates conflicts with active admin. He had another controller (ApiController) with this same DeviseTokenAuth inclusion already, which is the one he uses for endpoints, so he had to remove this line here in order to fix the issue he was having with active admin.

Here's a thread about the same issue Lean was having:

https://github.com/lynndylanhurley/devise_token_auth/issues/47

**Review by sebastiancaraballo — COMMENTED** [review]
Please add Trello card link and include a preview on the PR description please

File: db/seeds.rb (hunk 2 above)
Comment (JulianPasquale) [reply db/seeds.rb]: I don't think this can be done by accident actually. This is a task that needs to be manually run. Also seeds should be idempotent

File: app/controllers/application_controller.rb (hunk 3 above)
Comment (JulianPasquale) [reply app/controllers/application_controller.rb:2]: ah so this is in the ApiController as well. If that's the case then it's ok to remove this 👍

File: db/migrate/20220701173654_create_active_admin_comments.rb (hunk 1 above)
Comment (AgostinaDev) [reply db/migrate/20220701173654_create_active_admin_comments.rb]: I'm not sure if active admin comments are necessary here, I think you can disable them in the configuration file

File: config/initializers/active_admin.rb [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+  #
+  # Default:
+  # config.root_to = 'dashboard#index'
+
+  # == Admin Comments
+  #
+  # This allows your users to comment on any resource registered with Active Admin.
+  #
+  # You can completely disable comments:
+  # config.comments = false
```
Comment (AgostinaDev) [inline config/initializers/active_admin.rb]: You can use this configuration to disable comments

Comment (LeandroSoria2) [reply config/initializers/active_admin.rb]: I will change it on next commit

**Review by sebastiancaraballo — APPROVED** [review]

File: db/migrate/20220701173654_create_active_admin_comments.rb (hunk 1 above)
Comment (JulianPasquale) [reply db/migrate/20220701173654_create_active_admin_comments.rb]: If we are not going to use ActiveAdmin comments, then we need to remove this migrations as well

Comment (AgostinaDev) [reply db/migrate/20220701173654_create_active_admin_comments.rb]: I agree, this migration should be removed now that you've disabled comments in the configuration file @LeandroSoria2

Comment (LeandroSoria2) [reply db/migrate/20220701173654_create_active_admin_comments.rb]: I changed in last commit

File: db/schema.rb [hunk 5]
```diff
@@ -10,10 +10,28 @@
...
+    t.string "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["email"], name: "index_admin_users_on_email", unique: true
+    t.index ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true
+  end
+
+  create_table "topics", force: :cascade do |t|
```
Comment (AgostinaDev) [inline db/schema.rb]: Isn't this change supposed to be in another PR?

**Review by AgostinaDev — APPROVED** [review]
💪
